django_lightweight_queue/backends/reliable_redis.py

```diff
@@ -41,7 +42,16 @@ def startup(self, queue):
             'django_lightweight_queue:{}:processing:*'.format(queue),
         )
 
-        processing_queue_keys = self.client.keys(pattern)
+        # Work out which processing queues no longer have associated workers.
+        # Without this the startup process can end up racing against workers on
+        # other machines which are validly re-populating their processing queues
+        # as they work on jobs.
+        current_processing_queue_keys = set(self.client.keys(pattern))
```

Comment:
Maybe for a separate PR but should we consider using `SCAN` here? I doubt it would impact the startup of any given worker in isolation but it might help diminish startup time when multiple workers are starting up.

Reply:
We probably should look at that at some point, though I'm going to leave that for now.